### ISSUE TITLE
ui5-carousel: Arrow right and arrow left icons-only buttons are unlabelled

### ISSUE THREAD

**Issue by elenastoyanovaa** [SAP/ui5-webcomponents#2602]
Hello @SAP/ui5-webcomponents-topic-rd ,

Could you check this issue?

**Describe the bug**
In order to have a better screen reader support, each element should have a calculated accessible name. Currently the icon-only buttons, used for navigation between pages in the carousel web component are unlabelled.

**To reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/master/playground/components/Carousel/
2. Inspect the DOM of both ui5-buttons - no aria-label and title is set

**Expected behavior**
Similar to https://sapui5.hana.ondemand.com/#/entity/sap.m.Carousel/sample/sap.m.sample.Carousel add aria-label and title to both ui5-button with the following texts - "Previous Page", "Next Page" (see the Accessibility Status Wiki of the Carousel component). After implementation, please update the wiki.

**Affected components**
ui5-carousel
